io/buffer.ts

```diff
@@ -168,15 +167,14 @@ export class Buffer {
     this.#off = 0;
     this.#reslice(Math.min(m + n, MAX_SIZE));
     return m;
-  };
-
-  /** Grows the buffer's capacity, if necessary, to guarantee space for
+  } /** Grows the buffer's capacity, if necessary, to guarantee space for
    * another `n` bytes. After `.grow(n)`, at least `n` bytes can be written to
    * the buffer without another allocation. If `n` is negative, `.grow()` will
    * throw. If the buffer can't grow it will throw an error.
    *
    * Based on Go Lang's
    * [Buffer.Grow](https://golang.org/pkg/bytes/#Buffer.Grow). */
+
   grow(n: number): void {
```

Comment:
Intentional?

Reply:
No, `deno fmt` mangled it, I know how to reproduce it so will open an issue.